library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/focals/RectanglePromptFocal.java

```diff
@@ -170,6 +182,9 @@ public void prepare(@NonNull PromptOptions options, float targetX, float targetY
             mBaseBounds.bottom = targetY + halfHeight + mPadding;
             mBaseBoundsCentre.x = targetX;
             mBaseBoundsCentre.y = targetY;
+
+            mPath = new Path();
+            mPath.addRoundRect(mBaseBounds, mRx, mRy, Path.Direction.CW);
```

Comment:
Woops, a bit of an oversight by myself here, this should be in the update method. Now the prompt focus isn't pulsing :(